src/bootstrap/bootstrap.py

```diff
@@ -882,6 +882,10 @@ def build_bootstrap(self, color, verbose_count):
 
         # preserve existing RUSTFLAGS
         env.setdefault("RUSTFLAGS", "")
+        # FIXME: remove this if condition on the next bootstrap bump
+        # cfg(bootstrap)
+        if self.build_triple().startswith('mips'):
+            env["RUSTFLAGS"] += " -Ctarget-feature=+xgot"
```

Comment:
This should ideally have a comment why this exists.

Reply:
Fixed.